### ISSUE TITLE
can it do obj to tbl ?

### ISSUE THREAD

**Issue by monkeybee11** [Thutmose/tbl2obj#1]
know its not a issue but ive not no other way to ask is there a way to convert my obj model in to a tbl so im able to use it in my mc mod?

**Comment by Thutmose** [comment]
No, obj is more complex model format than tbl, so can't really convert other way, why can't you just use the obj in your mc mod? I made this for converting tbl to obj for use in minecraft mods.

**Comment by monkeybee11** [comment]
didt know thats a thing im new modding and coding in general
ive been using a program called mcreator to make my mod

**Comment by monkeybee11** [comment]
sry for dubble posting my phone dosent like the edit buttion  mcreator alows obj files to be used for blocks but for mobs u need to use .tbl or .java

**Comment by Thutmose** [comment]
I made my own animation/model system for using x3d/smd models for mobs, so I use this to convert to obj, then import to blender for animation, etc, then export from there for use with mobs

**Comment by Thutmose** [comment]
I think you can write code to use forge's obj models for mobs though.

**Comment by monkeybee11** [comment]
ok thx for the tip ill try looking in to this ill beable to use google blocks to make mc models if it works
